src/dbnode/storage/bootstrap.go

```diff
@@ -358,29 +325,25 @@ func (m *bootstrapManager) bootstrap() error {
 		if !ok {
 			err := fmt.Errorf("missing namespace from bootstrap result: %v",
 				id.String())
-			i := m.opts.InstrumentOptions()
-			instrument.EmitAndLogInvariantViolation(i, func(l *zap.Logger) {
-				l.Error("bootstrap failed",
-					append(logFields, zap.Error(err))...)
-			})
+			instrument.EmitAndLogInvariantViolation(m.instrumentation.opts.InstrumentOptions(),
+				func(l *zap.Logger) {
+					l.Error("bootstrap failed",
+						append(instrCtx.logFields, zap.Error(err))...)
+				})
```

Comment:
Actually, this whole anonymous fn could be a method on `instrCtx`, this would really unclutter the call site (same. goes for other `EmitAndLogInvariantViolation` calls as well).